### ISSUE TITLE
G8TesseractDelegate not found

### ISSUE THREAD

**Issue by hhimanshu** [gali8/Tesseract-OCR-iOS#141]
Hey 

I am trying to use this project and here is how it looks on my `Xcode Version 5.1 (5B130a)`  

![screen shot 2015-01-27 at 11 42 21 am](https://cloud.githubusercontent.com/assets/6589036/5925630/c898bff2-a619-11e4-9050-c4322720e144.png)

I used Option 1 to install

```
➜  ImageTest  pod install
Analyzing dependencies

CocoaPods 0.36.0.beta.1 is available.
To update use: `gem install cocoapods --pre`
[!] This is a test version we'd love you to try.

For more information see http://blog.cocoapods.org
and the CHANGELOG for this version http://git.io/BaH8pQ.

Downloading dependencies
Installing TesseractOCRiOS (3.4.0)
Generating Pods project
Integrating client project

[!] From now on use `ImageTest.xcworkspace`.
➜  ImageTest  
```

What's wrong here? What am I missing?

**Comment by kevincon** [comment]
See #111. Basically, we've made lots of changes since pod 3.4.0 (including changing the names of things like delegates and classes) and updated the documentation for those changes, but we haven't yet released a CocoaPod update (see the progress on our next release here, we're almost ready to release it: #123).

If you want to use the code in the repo right now (which will work for the provided examples, including `G8TesseractDelegate`, just use this in your Podfile instead:

```
pod 'TesseractOCRiOS', :git => 'https://github.com/gali8/Tesseract-OCR-iOS.git'
```

Otherwise, you can look at the older version of the Wiki's "Using Tesseract" page: https://github.com/gali8/Tesseract-OCR-iOS/wiki/Using-Tesseract-OCR-iOS/041e3dc5c7fef549aac27808fea968994696096d (notice how it uses `TesseractDelegate` instead of `G8TesseractDelegate`, which will work for pod version 3.4.0)

**Comment by kevincon** [comment]
Fixed by #123.
